### DISCUSSION TITLE
使用不同模型的說明及教學

### DISCUSSION THREAD

**Discussion by bloomin1** [GaiZhenbiao/ChuanhuChatGPT#575] [General]
有使用不同模型的說明及教學嗎?
另外什麼是雙欄PDF?

**Comment by Keldos-Li** [comment]
1. 会写的……吧。别急（或许社区愿意帮忙写的话也很好
2. 指的是你上传的pdf是不是一个双栏的pdf，是的话会用另外一个模块处理一下（

**Comment by Keldos-Li** [comment]
教程写好了~
https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/使用教程#使用本地-llm-模型
